core/jobs/batch_jobs/exp_proto_size_in_byte_addition_jobs.py

```diff
@@ -0,0 +1,65 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Jobs for adding proto_size_in_bytes attribute to exploration."""
+
+from __future__ import annotations
+
+from core import feconf
+from core.jobs import base_jobs
+from core.jobs.io import ndb_io
+
+import apache_beam as beam
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import exp_models
+
+(exp_models,) = models.Registry.import_models([models.NAMES.exploration])
+
+
+class AddProtoSizeInBytesToExplorationJob(base_jobs.JobBase):
```

Comment:
@vojtechjelinek Could you please provide some initial feedback on this beam job? 
Please ignore the formatting errors. 

Steps I am trying to do here:
1. Get all the Exploration model
2. Migrate to version 55
3. Write back to datastore